devcontainer/Dockerfile

```diff
@@ -84,5 +84,16 @@ RUN go install golang.org/x/tools/gopls@latest && \
     go install github.com/go-delve/delve/cmd/dlv@latest && \
     go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest && \
     go install gotest.tools/gotestsum@latest
+
```

Comment:
Just out of curiousity: don't we have a habit of just having a shell script? Since the block is the same across several  different Dockerfiles I would rather reuse a shell script and do something like:
`RUN --mount=type=bind,src=install_bazelisk.sh,target=/tmp/install_bazelisk.sh \
    chmod +x /tmp/install_bazelisk.sh && /tmp/install_bazelisk.sh`

This way you don't need to `COPY` that script it into the layer

Reply:
I like it... but will leave it to a subsequent PR dedicated to that refactoring, if that's fine for you. (I don't pretend I'll do it right after, nor whether I'll be the one doing it :wink:)